provider/pkg/client/client_test.go

```diff
@@ -365,3 +372,48 @@ func (m *mockAPI) GetResourceRequestStatus(ctx context.Context, requestToken str
 func (m *mockAPI) WaitForResourceOpCompletion(ctx context.Context, pi *types.ProgressEvent) (*types.ProgressEvent, error) {
 	return m.WaitForResourceOpCompletionFunc(ctx, pi)
 }
+
+func TestGetResourceRetryNotFoundRetrySettings(t *testing.T) {
+	ci := &clientImpl{}
+	millisecondDelays := []float64{}
+	attempts, backoff := ci.getResourceRetryNotFoundRetrySettings()
+	for attempt := 0; attempt < attempts; attempt++ {
+		delay, err := backoff.BackoffDelay(attempt, nil)
+		require.NoError(t, err)
+		millisecondDelays = append(millisecondDelays, float64(delay/time.Millisecond))
+	}
+	autogold.Expect([]float64{853, 437, 3000}).Equal(t, millisecondDelays)
```

Comment:
These are millisecond values for the attempt 0, 1, 2 suggested by backoff. Jitter makes them random in prod, but in testing the randomness is eliminated (see below).